docs/developer/centrifuge-chain/contributing/index.md

```diff
@@ -0,0 +1,223 @@
+```
+🏋️  Runtime size:             1.819 MB (1,906,886 bytes)
+🗜  Compressed:               Yes, 78.50%
+✨  Reserved meta:            OK - [6D, 65, 74, 61]
+🎁  Metadata version:         V14
+🔥  Core version:             centrifuge-1025 (centrifuge-1.tx2.au1)
+🗳️  system.setCode hash:      0x210cdf71ee5c5fbea3dc5090c596a992b148030474121b481a856433eb5720f3
+🗳️  authorizeUpgrade hash:    0x319e30838e1acc9dd071261673060af687704430c45b14cdb5fc97ec561e2a12
+🗳️  Blake2-256 hash:          0xb7f74401c52ee8634ad28fe91e8a6b1debb802d4d2058fdda184a6f2746477f6
+📦  IPFS:                     https://www.ipfs.io/ipfs/QmS3GDmbGKvcmSd7ca1AN9B34BW3DuDEDQ1iSLXgkjktpG
+```
+4. The `Blake2-256` hash should match the hex of the `authorizeUpgrade` call.
+    See more [here](docs/runtime-upgrade.md).
+
+## Generate new Spec and Parachain files
+
+This script will take a valid chain-spec chain_id and a flag to build new spec or not, and will output genesis spec (raw
+and plain), wasm and state files.
+
+```shell
+./scripts/export_parachain_files.sh demo true
+```
+Adapt parameters accordingly.
+
+
+## Benchmarking
+
+### Benchmarking runtimes
+
+When benchmarking pallets, we are just running the benchmarking scenarios they specify
+within their mocked runtime. This fails to actually benchmark said pallet in the context
+in which it will be actually used in production: within a specific runtime and composing
+with other pallets.
+
+To cover that, we run test for every pallet for a given runtime and use the output weights
+in production since those are the most trustworthy weights we can use.
+
+Note: This command should be run in a cloud environment that mimics closely the specs of
+the collator node the parachain will be running on.
+
+```shell
+./scripts/runtime_benchmarks.sh <runtime>
+```
+
+## Updating to a newer version of Polkadot
+
+When a new version of Polkadot is released, companion releases happen for the other
+parity projects such as the Polkadot SDK, as well as for other third-party projects
+such as the `ORML` pallets, `xcm-simulator`, etc.
+
+Therefore, updating this repository to a new version of Polkadot means updating all of these dependencies
+(internal and external to Centrifuge ) and have them all aligned on the same version of Polkadot.
+
+_Note: When we say "new version of Polkadot", we implicitly mean "Polkadot SDK, Frontier, ORML"._
+
+The high level flow to upgrade to a newer version of Polkadot is:
+
+1. Update all the Centrifuge-Chain dependencies to a revision that also points to the last version of Polkadot
+2. Fix all the breaking changes introduced by the latest versions
+
+### Update dependencies
+
+Since the Centrifuge Chain repository uses workspace inheritance, dependency revisions only need to be configured in the root `Cargo.toml`.
+
+1. **Update the `cargo patch` rules in `Cargo.toml`**
```

Comment:
```suggestion
1. **Update the `patch` rules section in `Cargo.toml`**
```

NIT. I think `cargo patch` leads to use a patch command for cargo, when it's a section in the `.toml` you need to change manually.